features/hooks.feature

```diff
@@ -20,108 +20,86 @@ Feature: hooks
       use Behat\Behat\Context\Context;
       use Behat\Gherkin\Node\PyStringNode,
           Behat\Gherkin\Node\TableNode;
+      use Behat\Hook\AfterFeature;
+      use Behat\Hook\BeforeFeature;
+      use Behat\Hook\BeforeScenario;
+      use Behat\Hook\BeforeStep;
+      use Behat\Step\Given;
+      use Behat\Step\Then;
 
       class FeatureContext implements Context
       {
           private $number;
           private $scenarioFilter;
 
-          /**
-           * @BeforeFeature
-           */
+          #[BeforeFeature]
```

Comment:
We might need separate `hooks_attributes.feature` and `hooks_annotations.feature`?